aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -124,13 +127,48 @@ def schedule_update_mirror(self):
     def update_mirror(self):
         self.write_config()
         try:
-            run_cmd(['apt-mirror', 'mirror.conf'], cwd=self.basepath)
+            run_cmd(['apt-mirror', 'mirror.conf'], cwd=self.basepath, logger=self.logger)
         finally:
             Mirror.objects.filter(id=self.id).update(refresh_in_progress=False)
 
     def user_can_modify(self, user):
         return user == self.owner
 
+    @property
+    def logger(self):
+        logpath = self.logpath()
+
+        logger = logging.getLogger('mirrorsvc.update_mirror.%s' % self.uuid)
+        logger.setLevel(logging.DEBUG)
+
+        for handler in logger.handlers:
+            logger.removeHandler(handler)
+
+        formatter = logging.Formatter('%(asctime)s: %(message)s')
+        logfp = logging.FileHandler(logpath)
+        logfp.setLevel(logging.DEBUG)
+        logfp.setFormatter(formatter)
+
+        logger.addHandler(logfp)
+
+        return logger
+
+    def logfilename(self):
+        LOG.debug('Determining config logfile name for mirror %s.' % (self.uuid))
+        print('Determining config logfile name for mirror %s.' % (self.uuid))
```

Comment:
Let's try to rely on the logging framework to get information out and not use print.